certbot-apache/certbot_apache/tests/augeasnode_test.py

```diff
@@ -145,11 +145,11 @@ def test_set_parameters_delete(self):
         self.assertTrue(found)
 
     def test_delete_child(self):
-        listens = self.config.parser_root.find_directives("Listen")
+        listens = self.config.parser_root.primary.find_directives("Listen")
         self.assertEqual(len(listens), 1)
         self.config.parser_root.primary.delete_child(listens[0])
 
-        listens = self.config.parser_root.find_directives("Listen")
+        listens = self.config.parser_root.primary.find_directives("Listen")
         self.assertEqual(len(listens), 0)
```

Comment:
I changed these to only interact with the `primary` because the other implementation will not be able to `delete_child` on a non-shallow child, so the checks in `dualparser.py` will fail.

Reply:
That's fine, thanks for clearing that up.